modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelbuilding/v0.1.2.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/emodelbuilding/v0.1.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
```

Comment:
All referenced schemas should be imported.

Reply:
done